app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_org = command.split()[1]
+    file_copy = command.split()[2]
```

Comment:
make ur code dry, u can unpack this to variables like this `command, file_org, file_copy = command.split()`